src/components/Footer.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="footer__container d-flex flex-row justify-content-end mx-5 mt-3">
+    <div class="footer-items__container d-flex flex-row my-auto mx-3">
+      <a class="footer__item d-flex flex-column align-items-center justify-items-center" ref="telegram" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Telegram" href="https://t.me/dnjco" target="_blank">
+        <img class="m-auto" src="@/assets/icons/telegram.svg" alt="">
+      </a>
+      <a class="footer__item d-flex flex-column align-items-center justify-items-center" ref="email" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Email" href="mailto:hi@dnj.co.ir" target="_blank">
+        <img class="m-auto" src="@/assets/icons/mail.svg" alt="">
+      </a>
+      <a class="footer__item d-flex flex-column align-items-center justify-items-center" ref="call" data-bs-toggle="tooltip" data-bs-placement="top" data-bs-title="Call" href="tel:03134420301" target="_blank">
+        <img class="m-auto" src="@/assets/icons/phone.svg" alt="">
+      </a>
+    </div>
+    <p class="footer__title iransans my-auto">راه های ارتباطی:</p>
+  </div>
+</template>
+
+<script lang="ts">
+import { Tooltip } from "bootstrap";
+import { defineComponent } from "vue";
+
+export default defineComponent({
+  setup() {},
+  mounted() {
+    new Tooltip(this.$refs.telegram)
```

Comment:
typing error in typescript.
test your code before you send PR.

Reply:
Fixed